### ISSUE TITLE
Could not find generator auditable:migration.

### ISSUE THREAD

**Issue by mxey** [harley/auditable#3]
(after adding auditable to the bundle)

```
mxey@mxey-mba42 ~/Code/satq (git)-[master] % rails generate auditable:migration
Could not find generator auditable:migration.
```

**Comment by harley** [comment]
arghh yes thanks! probably caused by me refactoring generators folder out hence rails didn't load the generator. should be fixed in a few moments. thanks again!

**Comment by harley** [comment]
@mxey can you check out version 0.0.8 now? it should also fix the other issue (#4) which was caused by the missing migration

**Comment by mxey** [comment]
Working now, thanks a lot :)
